data/mods/Dark-Skies-Above/monsters/wild_aliens/strays.json

```diff
@@ -117,52 +159,157 @@
+    "id": "dks_mon_stray_predator",
     "type": "MONSTER",
-    "name": "stray bruiser",
-    "description": "Swollen with muscle beyond what the human form would have normally allowed, this festering mass of raw power pulverizes whatever it comes across with the thick cudgels that were once its hands.  Its evolved regenative capacity seems to have granted it an outer shell now a protective layer of scar tissue, like a cocoon.  Tiny eyes peer out from behind the thick layers of callused flesh on its face, but its ears are now large holes in the sides of its head, suggesting it now has other ways of finding you.",
+    "name": { "str": "stray predator" },
+    "description": "Lithe muscle and pulasting crystal fused together, this creature crawls on four, grossly enlongated limbs sharpened to deadly points, spearing intruders to its domain.  Though it moves quickly, it seems only a matter of time before the very shell that protects it weighs it down to the ground.",
     "default_faction": "stray",
-    "looks_like": "mon_skeleton_brute",
+    "looks_like": "mon_zombie_predator",
     "bodytype": "human",
     "categories": [ "ALIEN" ],
-    "species": [ "ZOMBIE", "HUMAN" ],
-    "diff": 2,
+    "species": [ "BIOCRYSTAL", "HUMAN" ],
     "volume": "62500 ml",
     "weight": "81500 g",
     "hp": 100,
-    "speed": 90,
+    "speed": 110,
+    "material": [ "flesh" ],
+    "symbol": "S",
+    "color": "light_gray_red",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 6,
+    "melee_dice": 4,
+    "melee_dice_sides": 5,
+    "melee_cut": 3,
+    "dodge": 4,
+    "vision_night": 5,
+    "harvest": "mutant_human",
+    "path_settings": { "max_dist": 2 },
+    "special_attacks": [
+      [ "STRETCH_ATTACK", 20 ],
+      [ "LONGSWIPE", 20 ],
+      {
+        "type": "bite",
+        "cooldown": 5,
+        "damage_max_instance": [ { "damage_type": "stab", "amount": 10, "armor_multiplier": 0.5 } ]
+      },
+      [ "LUNGE", 20 ]
+    ],
+    "death_drops": "mon_zombie_hulk_death_drops",
+    "upgrades": { "half_life": 17, "into": "dks_mon_crystal_whip" },
+    "death_function": [ "NORMAL" ],
+    "flags": [
+      "SEES",
+      "HEARS",
+      "SMELLS",
+      "KEENNOSE",
+      "WARM",
+      "BASHES",
+      "POISON",
+      "PUSH_MON",
+      "PATH_AVOID_FALL",
+      "PATH_AVOID_FIRE",
+      "FILTHY",
+      "BLEED"
+    ]
+  },
+  {
+    "id": "dks_mon_stray_heavy",
+    "type": "MONSTER",
+    "name": { "str": "stray bruiser" },
+    "description": "Standing much steadier than its peers, this formerly human body is laden with thick crystal growths that pulsate as if alive.  Its hands are little more than spiked clubs now, dragging behind it as it trudges along.",
+    "default_faction": "stray",
+    "looks_like": "mon_zombie_tough",
+    "bodytype": "human",
+    "categories": [ "ALIEN" ],
+    "species": [ "BIOCRYSTAL", "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 80,
```

Comment:
Shouldn't these be larger than the normal stray?

Reply:
In retrospect, agreed. I'll make the crystaltender bigger to compensate (roughly the size of a gas zombie, which is slightly bigger than a boomer, which it was originally based on).